config/config.go

```diff
@@ -11,11 +10,6 @@ var Version string
 // Used so that we can generate fixed timestamps in tests
 var Clock TimestampGenerator = RealTimestampGenerator{}
 
-// Port that the local Broadcaster runs on
-const DefaultBroadcasterPort = 8935
-
-var DefaultBroadcasterURL = fmt.Sprintf("http://127.0.0.1:%d", DefaultBroadcasterPort)
```

Comment:
What you've done works well and is the more textbook way to do this - over time I've been leaning more towards the "if it never changes at runtime (or in this case after CLI parsing), just stick it in a global variable and avoid having to pass clients down through the full stack of method signatures" school of thought but I don't feel too strongly about that